### PR TITLE
chore: add user stories to sidebar

### LINKED ISSUE #195: Create new sidebar section for user stories
### Describe the proposed change(s).

Todo:

- [x] Wait until PR https://github.com/renovatebot/renovate/pull/16809 is merged
- [x] Wait until Renovate has updated the Git submodule with the contents of that PR
- [x] Create a section for User Stories in the `nav` section of the `mkdocs.yml` config file
- [x] Check sidebar works with `make build && make serve`
- [x] Open PR

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Add the new User Stories section to our sidebar

## Context:

Closes #195.

### CODE REVIEW

**Comment by rarkins** [comment]
I'd prefer it to be All Other > User Stories. Otherwise it will bloat our sidebar if we add many more

File: mkdocs.yml [hunk 1]
```diff
@@ -193,6 +193,8 @@ nav:
       - 'Schedule Presets': 'presets-schedule.md'
       - 'Workaround Presets': 'presets-workarounds.md'
   - All Other:
+      - User Stories:
+          - 'Swissquote': 'user-stories/swissquote.md'
```
Comment (rarkins) [inline mkdocs.yml:197]: We can start like this but longer term I don't think we should have each story in the sidebar

Comment (HonkingGoose) [reply mkdocs.yml:197]: Can you explain more? I don't understand why you think this. 😉

Comment (rarkins) [reply mkdocs.yml:197]: Let's say we have 5-10 user stories. We don't want the sidebar to be too long. Or are third level items collapsed by default?

Comment (HonkingGoose) [reply mkdocs.yml:197]: The "User Stories" part is collapsed by default, you must click to expand it.

**Review by rarkins — APPROVED** [review]

**Pull request merged**